trustwallet/wallet-core
# Add own implementation of crc32 checksum, remove boost dependancy

## Description

This PR adds a custom implementation of the crc32 cyclical redundancy check and by doing so removes the dependancy upon boost::crc (part of issue #2540 ). 

## How to test

Run tests and see if they fail. If the implementation were incorrect, several Cardano tests would fail as it is used there.

## Types of changes

* Improvement: This code introduces no features and should be fully backwards-compatible.

## Checklist
